src/parser/position.rs

```diff
@@ -0,0 +1,77 @@
+use crate::parser::Span;
+#[cfg(not(target_arch = "wasm32"))]
+use proptest::prelude::*;
+
+/// Source code position of an expression in a file
```

Comment:
In general, this implementation seems very incomplete. I’m fine with deferring completion, but there should be a module comment that makes the draft status clear and refers to an open issue for completing.

Reply:
Having skimmed this and the comments, I wonder if we can just leave this all out. Are we getting any benefit from including this in its current state? If not, we could save inclusion for when it's a bit more complete perhaps.

Reply:
Okay, some context: `Pos` is a direct port of https://github.com/lurk-lab/yatima-lang-alpha/blob/main/core/src/position.rs, and this technique  been *very* useful to me over both in writing/debugging parsers, and also generating pretty user error messages.

Currently `Pos` isn't hooked up to generate user relevant error messages yet, but it's still useful as a developer debug feature. It also doesn't affect anything other than debugging (per the above comments) so removing it would be unnecessary and really not ideal in my view